.github/scripts/utils.py/check-services.py

```diff
@@ -265,25 +208,47 @@ def main():
             continue
 
         service_type = service.get("recommended", {}).get("output", "rtmp_output")
-        if service_type not in {"rtmp_output", "ffmpeg_hls_muxer", "ftl_output"}:
+        if service_type not in {"rtmp_output", "ffmpeg_hls_muxer"}:
             print("Unknown service type:", service_type)
             new_services["services"].append(service)
             continue
 
+        protocol = "rtmp" if service_type == "rtmp_output" else "hls"
+
         # create a copy to mess with
         new_service = service.copy()
         new_service["servers"] = []
 
         # run checks for all the servers, and store results in timestamp cache
-        for server in service["servers"]:
-            if service_type == "ftl_output":
-                is_ok = check_ftl_server(server["url"])
-            elif service_type == "ffmpeg_hls_muxer":
-                is_ok = check_hls_server(server["url"])
-            else:  # rtmp
-                is_ok = check_rtmp_server(server["url"])
+        try:
+            servers = [s["url"] for s in service["servers"]]
+            tasks = []
+            for host in check_servers:
+                tasks.append(
+                    asyncio.create_task(
+                        check_servers_task(session, host, protocol, servers)
+                    )
+                )
+            results = await asyncio.gather(*tasks)
+        except Exception as e:
+            print(
+                f"❌ Querying server status for \"{service['name']}\" failed with: {e}"
+            )
+            return 1
+
```

Comment:
Could we add perhaps a 2 second wait between services in order to add a rudimentary rate limit to the API requests?

(This comment's position in the diff is not of relevance)